src/runtime/core/logger/rotate_file_logger_backend.cc

```diff
@@ -87,6 +100,35 @@ void RotateFileLoggerBackend::Initialize(YAML::Node options_node) {
   }
   formatter_.SetPattern(pattern_);
 
+  // if enable_sync, set sync timer
+  if (options_.enable_sync) {
+    // if enable_sync, sync_executor_name must be set
```

Comment:
Comments explaining the code are too verbose; the code itself should express its meaning